products/cockroachdb.md

```diff
@@ -0,0 +1,130 @@
+---
+title: CockroachDB
+category: database
+iconSlug: cockroachlabs
+permalink: /cockroachdb
+alternate_urls:
+-   /cockroach
+-   /crdb
+versionCommand: cockroach version
+releasePolicyLink: https://www.cockroachlabs.com/docs/releases/release-support-policy
+changelogTemplate: https://www.cockroachlabs.com/docs/releases/v__RELEASE_CYCLE__
+releaseDateColumn: true
+activeSupportColumn: Maintenance
+eolColumn: Assistance
```

Comment:
```suggestion
activeSupportColumn: Maintenance Support
eolColumn: Assistance Support
```

As defined in https://www.cockroachlabs.com/docs/releases/release-support-policy.